src/main/resources/com/microsoft/azure/util/AzureImdsCredentials/credentials.jelly

```diff
@@ -7,6 +7,11 @@
         <f:textbox/>
     </f:entry>
 
+    <f:entry title="${%Client ID}" field="clientId"
```

Comment:
Could try adding a `credentials.properties` file with:

`Client ID=Client ID (optional)`

that would show that it's not needed in most cases